### ISSUE TITLE
InfinityLib duplicates vanilla keys to registery

### ISSUE THREAD

**Issue by MissPotato** [InfinityRaider/InfinityLib#47]
Log: https://mclo.gs/3m1Y37c

The only mod installed is InfinityLib v2.1.2 on forge version 40.1.51. While this doesn't seem to break anything, worlds load fine, I assume this isn't intentional.

Specifically it seems to be duplicating all "ResourceKey[minecraft:worldgen/noise /" keys.

**Comment by InfinityRaider** [comment]
Interesting, I am not interacting with those at all, but I'll have a look

**Comment by InfinityRaider** [comment]
I am failing to reproduce this...

**Comment by MissPotato** [comment]
I've tried using different launchers, I've tried changing forge versions, I've tried changing java versions, whole new instances, in every situation I get duplicate keys. I have even tried downgrading Infinity lib to 2.1.0 and it happens with that too.  I updated to 2.1.3 and it still happens. 

https://paste.ee/p/84BLM

My java version is: OpenJDK Runtime Environment Temurin-17.0.3+7 (build 17.0.3+7)
Game version is 1.18.2
Forge is 40.1.51
LWJGL 3.2.2
